OrbitQt/servicedeploymanager.cpp

```diff
@@ -335,6 +336,25 @@ outcome::result<void> ServiceDeployManager::ConnectToServer() {
   return outcome::success();
 }
 
+void ServiceDeployManager::StartWatchdog(std::string_view start_passphrase,
+                                         int interval_in_ms) {
+  orbit_service_task_->Write(start_passphrase);
+
+  const auto watchdog_send_timer = QPointer(new QTimer());
```

Comment:
I think it makes more sense to put the timer into a member variable. No allocation necessary.

Reply:
If not the timeout callback might be called when the service deploy manager is already destructed.

Reply:
By the way the QPointer makes no sense here because watchdog_send_timer only exists for a short time.

Reply:
makes sense! done